tc/autotuner/genetic_search.cc

```diff
@@ -230,7 +230,7 @@ TuningConfiguration GeneticSearch::crossover(
     }
   };
 
-  for (size_t i = 0; i < kMutateIterations; ++i) {
```

Comment:
hmm, this was a bug!

Reply:
No, the code is actually correct. The outer induction variable is never used inside the body of the loop.